modules/examples/examples/zenoh_sub.cpp

```diff
@@ -19,30 +21,41 @@
 #include "hephaestus/ipc/zenoh/subscriber.h"
 #include "zenoh_program_options.h"
 
+namespace {
+volatile std::atomic_bool keep_running(true);  // NOLINT
+}  // namespace
+
 auto main(int argc, const char* argv[]) -> int {
+  std::ignore = std::signal(SIGINT, [](int signal) {
+    if (signal == SIGINT) {
+      LOG(INFO) << "SIGINT received. Shutting down...";
+      keep_running.store(false);
+    }
+  });
   try {
-    auto desc = getProgramDescription("Periodic publisher example");
+    auto desc = getProgramDescription("Periodic publisher example", ExampleType::Pubsub);
     const auto args = std::move(desc).parse(argc, argv);
 
     auto [session_config, topic_config] = parseArgs(args);
 
-    fmt::println("Opening session...");
-    fmt::println("Declaring Subscriber on '{}'", topic_config.name);
+    LOG(INFO) << "Opening session...";
+    LOG(INFO) << fmt::format("Declaring Subscriber on '{}'", topic_config.name);
 
     auto session = heph::ipc::zenoh::createSession(std::move(session_config));
 
     auto cb = [topic = topic_config.name](const heph::ipc::MessageMetadata& metadata,
                                           const std::shared_ptr<heph::examples::types::Pose>& pose) {
-      fmt::println(">> Time: {}. Topic {}. From: {}. Counter: {}. Received {}",
-                   std::chrono::system_clock::time_point{
-                       std::chrono::duration_cast<std::chrono::system_clock::duration>(metadata.timestamp) },
-                   metadata.topic, metadata.sender_id, metadata.sequence_id, *pose);
+      LOG(INFO) << fmt::format(
```

Comment:
nit. I think this is the output of the example so I would keep it as `fmt::println`